src/os/file.go

```diff
@@ -11,7 +11,8 @@ import (
 
 // Portable analogs of some common system call errors.
 var (
-	ErrUnsupported = errors.New("operation not supported")
```

Comment:
Note, changed initial capitalisation of this for consistency.  Can do it the other way around too if that'd be better. :wink: